lib/create-organization-api.js

```diff
@@ -193,6 +194,50 @@ export default function createOrganizationApi ({ http }) {
       .then(response => wrapTeamMembershipCollection(http, response.data), errorHandler)
   }
 
+   /**
+   * Get a Team Space Membership with given teamSpaceMembershipId
+   * @memberof ContentfulOrganizationAPI
+   * @param {string} teamSpaceMembershipId - Team ID
+   * @param {string} id - Team Membership ID
+   * @return {Promise<TeamMembership>} Promise for an Team Membership
+   * @example
+   * const contentful = require('contentful-management')
+   *
+   * client.getOrganization('organizationId')
+   * .then((organization) => organization.getTeamSpaceMembership('teamSpaceMembershipId'))
+   * .then((teamSpaceMembership) => console.log(teamSpaceMembership))
+   * .catch(console.error)
+   */
+  function getTeamSpaceMembership (teamSpaceMembershipId) {
```

Comment:
@guilebarbosa I am not able to comment not changed code, but you could add this function in the top of this file (line 26 or so), if you would like this ot appear in the overview of the functions the docs.

Reply:
Can you explain what you mean?